operator/v1/types_console.go

```diff
@@ -202,6 +205,52 @@ type AddPage struct {
 	DisabledActions []string `json:"disabledActions,omitempty"`
 }
 
+type PerspectiveState string
+
+// These are valid perspective states. "Enabled" means the perspective is shown.
+// "Disabled" means the Perspective is hidden.
+// "AccessReview" means access review check is required to show or hide a Perspective.
+const (
+	Enabled      PerspectiveState = "Enabled"
+	Disabled     PerspectiveState = "Disabled"
+	AccessReview PerspectiveState = "AccessReview"
+)
+
+// `required` and  `missing` can work together esp. in the case where cluster admin
+// wants to show another perspective to users without specific permissions. If both `required` and `missing`
+// are not specified for `state: AccessReview`, access review check will be ignored and perspective will be enabled.
```

Comment:
Is this intuitive? Both fields say they only show the field when their checks pass/fail respectively, so I would have thought if there's no check in the required (which would mean it's not shown) and there's no check in the missing (which would mean it's not shown), then the perspective would not be shown?

Two not shows shouldn't make a show should it?